config/locales/article/article-es.yml

```diff
@@ -1,12 +1,104 @@
 ---
 es:
   article:
+    article_links:
+      n_pages_refer_to:
+        one: 1 página se refiere a %{article}
+        other: "%{count} páginas se refieren a %{article}"
+      n_subjects_refer_to:
+        one: 1 sujeto se refiere a %{article}
+        other: "%{count} sujetos se refieren a %{article}"
+      show_pages_that_mention: Mostrar páginas que mencionen %{article} en todos los trabajos
     combine_duplicate:
       selected_subjects_combined: Asignaturas seleccionadas combinadas con %{title}
     delete:
       must_remove_referring_links: Debes eliminar todos los enlaces de referencia antes de eliminar este tema.
+    edit:
+      autolink: Enlace automático
+      categories: Categorías
+      combine_selected: Combinar seleccionado
+      confirm_delete_subject: "¿Está seguro de que desea eliminar este asunto? ¡Después de borrar no podrás recuperarlo!"
+      delete_subject: Eliminar asunto
+      description: Descripción
+      latitude: Latitud
+      longitude: Longitud
+      microphone: Micrófono
+      n_pages:
+        one: 1 pagina
+        other: "%{count} páginas"
+      no_duplicates_found: No se encontraron duplicados
+      no_duplicates_found_description: El tema es único dentro de la colección, no se han encontrado posibles duplicados del tema "%{article}".
+      possible_duplicates:
+        one: Posible duplicado
+        other: Posibles duplicados
+      possible_duplicates_description: Revise la lista a continuación y seleccione los temas para combinar. Los duplicados se reasignarán para que todos los enlaces existentes apunten al asunto "%{article}".
+      save_changes: Guardar cambios
+      select_categories: Seleccionar categorías
+      title: Título
+      uri: URI
+    graph:
+      dot:
+        title: Temas mencionados frecuentemente con %{subject}
+    list:
+      actions: Comportamiento
+      add_child_category: Agregar categoría secundaria
+      add_root_category: Agregar categoría raíz
+      categories: Categorías
+      category_back_message: Una vez que termine de editar las categorías, %{page_link} para volver a la página "%{page}".
+      click_here: haga clic aquí
+      confirm_delete_category: "¿Está seguro de que desea eliminar esta categoría y todas sus subcategorías? ¡Después de borrar no podrás recuperarlo!"
+      create_category_message: "%{link} que usarás para agrupar materias."
+      create_the_first_category: Crear la primera categoría
+      delete_category: Eliminar categoría
+      disable_gis_for_category: Deshabilitar GIS para la categoría
+      edit_category: Editar categoria
+      enable_gis_for_category: Habilitar SIG para la categoría
```

Comment:
```suggestion
      enable_gis_for_category: Habilitar GIS para la categoría
```

Reply:
Is GIS really "GIS" in Spanish?  Wikipedia uses the header [sistema de información geográfica (SIG)](https://es.wikipedia.org/wiki/Sistema_de_informaci%C3%B3n_geogr%C3%A1fica)

Reply:
From a little Googling, it looks like it is SIG in Spanish:
> Un Sistema de Información Geográfica (SIG o GIS, de sus siglas en inglés Geographical Information System) 

GIS is the English acronym.
(from https://geoinnova.org/blog-territorio/que-es-un-sig-gis-o-sistema-de-informacion-geografica/)